frontend/app/src/components/asset-manager/AssetForm.vue

```diff
@@ -525,6 +532,10 @@ export default class AssetForm extends Vue {
     }
     return identifier;
   }
+
+  private sanitizeAddress(address: string) {
+    return address.replace(/\s/g, '');
+  }
```

Comment:
Isn't it better to use trim here? https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/String/Trim

Reply:
Yes trim will do it. But I think, isn't address also cannot have `space` too in the middle, why dont remove it altogether? @yabirgb